src/App.js

```diff
@@ -1,5 +1,113 @@
-function App() {
-  return <div></div>;
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Heading from './components/Heading/Heading';
+
+const initialItems = [
```

Comment:
Once you have fetched the data, it is not really necessary to have a initial state anymore :)